src/main/java/com/agilefaq/codedoctorsample/App.java

```diff
@@ -8,6 +8,7 @@
  * Hello world!
  */
 public class App {
+    public int i = 11;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make i a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)